src/azure-cli/azure/cli/command_modules/resource/commands.py

```diff
@@ -171,15 +173,15 @@ def load_command_table(self, _):
         g.custom_command('list', 'list_locks')
         g.custom_show_command('show', 'get_lock')
         g.custom_command('update', 'update_lock')
-
+    from azure.cli.core.commands import DeploymentOutputLongRunningOperation
```

Comment:
Could we move `from azure.cli.core.commands import DeploymentOutputLongRunningOperation` under  `with self.command_group('resource', resource_custom, resource_type=ResourceType.MGMT_RESOURCE_RESOURCES) as g:`

Reply:
Maybe it will be better to combine it with CliCommandType at the top.